docs/book/releases/v2.3.0.md

```diff
@@ -0,0 +1,84 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.3.0 release
+
+## New Feature
+
+- CSI Migration for in-tree vSphere volumes is promoted to GA. This feature requires upgrading Kubernetes to 1.19 release.
+
+- XFS Filesystem support as an Alpha feature. Volumes can be provisioned by specifying `xfs` as the `fstype` in the [Storage Class](https://vsphere-csi-driver.sigs.k8s.io/features/block_volume.html).
+
+- Raw Block volume support as an Alpha feature. Visit the [Raw Block Volumes documentation](https://vsphere-csi-driver.sigs.k8s.io/features/raw_block_volume.html) for more details.
```

Comment:
Can we update https://vsphere-csi-driver.sigs.k8s.io/features/raw_block_volume.html and add a link to 2.3 release note page and also change a wordings 

"vSphere CSI Driver release v2.3.0 will have Raw Block Volume feature released as Alpha" => "vSphere CSI Driver release v2.3.0 has Raw Block Volume feature released as Alpha"

Reply:
done